Challenges/LINQ/LINQ.Challenge.Tests/IntermediateTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using FluentAssertions;
+
+namespace LINQ.Challenge.Tests;
+
+public class IntermediateTests
```

Comment:
Could we add a distinct here?

Reply:
Added to Intermediate - get list of unique numbers